src/dcommands/ccommand.js

```diff
@@ -124,8 +124,8 @@ class CcommandCommand extends BaseCommand {
                 + 'For more in-depth command customization, see the `editcommand` command.\n'
                 + '\n**Subcommands:**\n'
                 + `${subcommands.map(x => `**${x.name}**`).join(', ')}`
-                + '\nFor more information about a subcommand, do `b!cc help <subcommand>.`\n'
-                + '\nFor more information about BBTag, visit <https://blargbot.xyz/tags>.'
+                + '\nFor more information about a subcommand, do `b!cc help <subcommand>`.\n'
+                + '\nFor more information about BBTag, visit https://blargbot.xyz/tags.'
```

Comment:
i'd much rather this stay escaped

Reply:
Previously, the content was `<https://...>` which is currently displayed like this on the website :
![image](https://user-images.githubusercontent.com/32441291/46587402-42d09c00-ca8c-11e8-8b03-dfd18be971da.png)

Wanted to get rid of that problem so I removed the characters that were recognized by the parser.
I'm unsure if `\<https://...\>` would work or not but it's still the thing I've changed for your request.